fastlane/Fastfile

```diff
@@ -21,6 +21,9 @@ SCREENSHOT_DEVICES = [
   'iPad Pro (12.9-inch) (3rd generation)'
 ].freeze
 
+MAIN_BUNDLE_IDENTIFIERS = %[com.automattic.woocommerce]        # Registered in our main account, for development and AppStore
+ALPHA_BUNDLE_IDENTIFIERS = %[com.automattic.alpha.woocommerce] # Registered in our Enterprise account, for AppCenter / Installable Builds
```

Comment:
Contrary to WPiOS which have many bundleIDs because of its various App Extensions (Widgets, ShareExt, etc), WooCommerce currently only has that single one (at least to my knowledge).

I think that's why we never bothered to add that `register_new_device` lane in WCiOS before, as this felt not as necessary and could be easily done manually compared to WPiOS which has so many profiles to be updated.
Still, I feel this is still very useful to have even for a single bundleID, both because it makes it easy to scale this if we end up adding App Extensions to WCiOS one day, but also because that makes it a no-brainer just having to run the same lane for WCiOS as WPiOS, instead of wondering which of the various profiles in ADC would need to be updated when doing it manually 🙂